src/ImageGallery.jsx

```diff
@@ -1097,8 +1113,8 @@ export default class ImageGallery extends React.Component {
             [
               this.props.showNav &&
                 <span key='navigation'>
-                  {this.props.renderLeftNav(slideLeft, !this._canSlideLeft())}
-                  {this.props.renderRightNav(slideRight, !this._canSlideRight())}
+                  {this.props.renderLeftNav(isRTL ? slideRight : slideLeft, !this._canSlideLeft())}
+                  {this.props.renderRightNav(isRTL ? slideLeft : slideRight, !this._canSlideRight())}
```

Comment:
Sorry for not noticing this on my first review, but I found a bug with this when `infinite` is set to `false`.

You'll need to also swap the function call too.